ci/tell-slack-failed.yml

```diff
@@ -8,10 +8,13 @@ steps:
   - template: bash-lib.yml
     parameters:
       var_name: bash_lib
+    condition: and(failed(),
```

Comment:
can we just set this to `always` so we don’t have to keep the two in sync? The check for sending it below should be sufficient

Reply:
I suppose we can spare the extra second.